client/src/components/JurisdictionAdmin/BatchRoundSteps/EnterTalliesStep.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { toast } from 'react-toastify'
+import { Button } from '@blueprintjs/core'
+import { IJurisdiction } from '../../UserContext'
+import { IRound } from '../../AuditAdmin/useRoundsAuditAdmin'
+import BatchRoundDataEntry from '../BatchRoundDataEntry'
+import { useBatches, useFinalizeBatchResults } from '../useBatchResults'
+import { useConfirm, Confirm } from '../../Atoms/Confirm'
+import { StepPanel, StepActions } from '../../Atoms/Steps'
+import LinkButton from '../../Atoms/LinkButton'
+
+interface IEnterTalliesStepProps {
+  previousStepUrl: string
+  jurisdiction: IJurisdiction
+  round: IRound
+}
+
+const EnterTalliesStep: React.FC<IEnterTalliesStepProps> = ({
```

Comment:
@arsalansufi I thought you might want to see roughly what this component looks like so far since it will be rendering the list/detail batch auditing view.